src/main/java/org/broadinstitute/hellbender/utils/test/VariantContextTestUtils.java

```diff
@@ -70,10 +79,124 @@ static Object normalizeScientificNotation(final Object attribute){
         return attribute;
     }
 
+    public static VariantContext sortAlleles(final VariantContext vc, final VCFHeader header){
```

Comment:
Javadoc on this explaining the sorting and the fact that it relies on the correctness of the vcf header.